index.html

```diff
@@ -0,0 +1,371 @@
+							</h3>
+
+							<div>
+								<p class="text-light">
+									Appreciate various creations applying CC license of artists,
+									organized from Art Center Nabi
+								</p>
+							</div>
+						</div>
+
+						<div
+							class="d-flex justify-content-between align-content-center shadow p-2 mb-2 bgColor3"
+							id="program-info">
+							<div class="program-img">
+								<img
+									class="text-center"
+									src="./asset/images/program_forum.png"
+									alt="lecture" />
+							</div>
+
+							<h3
+								class="text-danger text-center px-2 display-5"
+								id="program-task">
+								Forum
+							</h3>
+
+							<p class="text-light">
+								Have the time to share your thoughts and opinions with experts
+								for each topic.
+							</p>
+						</div>
+
+						<div
+							class="d-flex justify-content-between align-content-center shadow p-2 mb-2 bgColor3"
+							id="program-info">
+							<div class="program-img">
+								<img
+									class="text-center"
+									src="./asset/images/program_workshop.png"
+									alt="lecture" />
+							</div>
+
+							<h3
+								class="text-danger text-center px-2 display-5"
+								id="program-task">
+								Work shop
+							</h3>
+
+							<p class="text-light">
+								Try creating your own work using open source license rather than
+								just watching at it.
+							</p>
+						</div>
+
+						<div
+							class="d-flex justify-content-between align-content-center shadow p-2 mb-2 bgColor3"
+							id="program-info">
+							<div class="program-img">
+								<img
+									class="text-center"
+									src="./asset/images/program_opportunities.png"
+									alt="lecture" />
+							</div>
+
+							<h3
+								class="text-danger text-center px-2 display-5"
+								id="program-task">
+								Ignite
+							</h3>
+
+							<p class="text-light">
+								get opportunities to network with CC affiliates around the
+								world, also after the summit.
+							</p>
+						</div>
+					</div>
+					<h4 class="see-program text-center m-1 py-2">
+						SEE THE WHOLE PROGRAM
+					</h4>
+					<div
+						class="shadow-sm p-2 mb-4 bg-danger mt-4 container"
+						id="join-program">
+						<h4 class="text-center text-white p-1 display-5">
+							Join Victoria Alumni 2023 Summit
+						</h4>
+					</div>
+				</section>
+				<!-- feature speaker -->
+				<section class="feature-speaker-container bgColor-2 py-2">
+					<div class="speaker-content container-fluid">
+						
+							<h4 class="mt-3 display-5 text-dark text-center">Feature Speaker</h4>
+						
+						<div class="d-flex justify-content-center"><hr class="bar bg-danger"> </hr></div>
+<!-- speakers mobile and desktop -->
+						<div class="feature-speaker-bio-mobile"></div>
+						<div class="feature-speaker-bio"> </div>
+
+						<div class="shadow p-1 mb-4 bg-white">
+							<p class="text-center textColor-dark" id="join-program">
+								more <span class="text-danger"> V </span>
```

Comment:
[REQUIRED]Kindly reduce and replace the `v` with a dropdown icon that looks like the one in the sample design. It is a requirement for your design to match the template

[REQUIRED] I noticed that your more button is not working Kindly fix it.